server/tests/src/test/java/org/infinispan/server/functional/hotrod/HotRodCacheQueries.java

```diff
@@ -287,13 +289,13 @@ public void testVectorSearch() {
       query.setParameter("a", center.getByteEmbedding());
       query.setParameter("k", 3);
       List<KeywordVector> list = query.list();
-      assertThat(list).extracting(KeywordVector::getName).containsExactly("bla-7", "bla-6", "bla-8");
+      assertThat(list).extracting(KeywordVector::getName).contains("bla-7", "bla-6", "bla-8");
```

Comment:
Hmm, interesting. I thought the order would be matter in this case?

Reply:
Guessing it doesn't order them across node responses. I am not sure if this is an issue or not.